packages/gitbook/src/components/SpaceLayout/SpaceLayout.tsx

```diff
@@ -17,15 +16,15 @@ import { CONTAINER_STYLE } from '@/components/layout';
 import { ColorDebugger } from '@/components/primitives/ColorDebugger';
 import { SearchModal } from '@/components/Search';
 import { TableOfContents } from '@/components/TableOfContents';
-import { ContentPointer, ContentTarget, SiteContentPointer } from '@/lib/api';
+import { ContentTarget, SiteContentPointer } from '@/lib/api';
 import { ContentRefContext } from '@/lib/references';
 import { tcls } from '@/lib/tailwind';
 
 /**
  * Render the entire content of the space (header, table of contents, footer, and page content).
  */
 export function SpaceLayout(props: {
```

Comment:
This is why I think the other one should be named `CustomizationRootLayout`, otherwise it's confusing with this component. This one has a valid name since it renders a space.